### ISSUE TITLE
Non-SKOS property labels defined in another graph are not displayed

### ISSUE THREAD

**Issue by tfrancart** [NatLibFi/Skosmos#669]
URL : http://erato.sparna.fr/authority/legal-institution-historique/fr/page/107

How to reproduce :

1. Go to above URL. Note how no non-skos properties are displayed
2. Click on "RDF/XML" and examine RDF/XML for the resource. Note there are "euvoc" properties being stated on the resource (euvoc:startDate, euvoc:endDate)
3. Go to http://data.legilux.public.lu/sparql and issue the following SPARQL query :

```
select distinct ?g ?p ?o
where {
GRAPH ?g {
      <http://publications.europa.eu/ontology/euvoc#startDate> ?p ?o .
}
}
```

Note how there are rdfs:label for the `euvoc:startDate` property defined in the graph `http://data.legilux.public.lu/ontology`.

At https://github.com/NatLibFi/Skosmos/blob/master/model/Concept.php#L333, there is the requirement that the property declaration belongs to the same RDF graph as the concept being displayed.

This has the consequence that, if this requirement is not met, the property label will not be found and the property will not be displayed (see few lines below : https://github.com/NatLibFi/Skosmos/blob/master/model/Concept.php#L341)

We follow a different (and I think rather common) pattern, where the ontology is loaded in its own, separate named graph. Many vocabularies, each in its own named graph, rely on that unique ontology declaration.

This could be solved by either :
1. querying for the property label in a graph-agnostic way (through GenericSparql::queryLabel ?)
2. add an extra parameter on the vocabulary config, indicating the named graph of the ontology (skosmos:ontologyNamedGraph ?)
3. Another idea ?

**Comment by osma** [comment]
Makes sense. I think option 1 would be preferable. Then one could load an ontology into a separate graph and it would just be used for property labels without any extra configuration.

**Comment by osma** [comment]
Want to do a PR again? :)

**Comment by tfrancart** [comment]
Sure
